app/views/jobseekers/profiles/personal_details/phone_number.html.slim

```diff
@@ -0,0 +1,22 @@
+- content_for :page_title_prefix, t(".page_title")
+
+- content_for :breadcrumbs do
+  = govuk_back_link text: t("buttons.back"), href: back_url
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    = form_for @step, url: { action: :update }, method: :post, as: :personal_details_form do |f|
+      = f.govuk_error_summary
+
+      span.govuk-caption-l = t(".caption")
+      h1.govuk-heading-l = t(".heading")
```

Comment:
these kind of things need to be legends on the fieldset or labels on the form(group)

Reply:
e.g on one of the fieldsets in my task `= f.govuk_fieldset legend: { text: t(".page_title"), tag: "h1", size: "l" } do`